app/models/spree/variant_decorator.rb

```diff
@@ -60,14 +60,11 @@ def google_base_item_color
     end
 
     private
+
     def warehouse_count_on_hand
-      warehouse_id = stock_locations.find_by(name: "Warehouse").id
-      for stock_item in stock_items
-        if stock_item.stock_location_id == warehouse_id
-          return stock_item.count_on_hand
-        end
-      end
+      warehouse_id = stock_locations.find_by(name: 'US Warehouse').id
```

Comment:
Can we leverage rails secrets here and leave `US Warehouse` as the default hardcoded value?

Reply:
The idea behind this being we don't need to update these names with a deploy?

Reply:
We'd still deploy but only change the secret + deploy and not this project + deploy. Slightly better?

Reply:
Uh right! No hash bump required in the Gemfile.